kbase-extension/static/kbase/js/widgets/narrative_core/catalog/catalog_util.js

```diff
@@ -0,0 +1,108 @@
+define([
+    'bluebird',
+    'jquery'
+], function (Promise, $) {
+    'use strict';
+
+    function CatalogUtil() {
+
+        // edited from: http://stackoverflow.com/questions/3177836/how-to-format-time-since-xxx-e-g-4-minutes-ago-similar-to-stack-exchange-site
+        this.getTimeStampStr = function (objInfoTimeStamp) {
```

Comment:
Eventually (in the magical after-contractor's-meeting time) we should get the kbase-ui and narrative util modules in sync.